manifester_settings.yaml.example

```diff
@@ -2,6 +2,7 @@
 log_level: "info"
 offline_token: ""
 proxies: {"https": ""}
+username_prefix: "example_username"  # replace value with a unique username
```

Comment:
While username_prefix is nice and specific, is there any other case you would want to use the username in the future? If so, I think just calling it `username` is fine.

Reply:
It's unlikely that I'll need to have a username setting, as the offline token/access token are the only auth credentials required for access to the API. However, I do want the name of the setting to clearly indicate that the username is being used as a textual identifier and not as a login credential, so I'd like to keep the `prefix` part there. 

Broker has a similar setting called `name_prefix`, which is slightly shorter. I can adopt that convention here if the length of the setting name is your concern. What do you think?

Reply:
IMO `username_prefix` would be more clear.

Reply:
Not a problem with the length or anything just trying to future proof the codebase.